redhat-cop/automate_tower_genie_collections
# Workflow schema

### What does this PR do?
Adds option for workflow schemas to be used instead of workflow nodes lope. 
uses [This Awx.awx PR](https://github.com/ansible/awx/pull/9309)


### How should this be tested?
Additinonal tests should be added to test this, I propose we use the tower_export data as it is already in this format, but as an additional test so we keep testing the now 3 ways to deploy schemas.
